styles.css

```diff
@@ -7,14 +7,43 @@ a {
   text-decoration: none;
 }
 
-#back-images-intro {
-  width: 100%;
-  height: 100%;
+.about {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  justify-content: center;
+  max-width: 100%;
+  padding-top: 48px;
+  background-color: #fff;
+  border-top-right-radius: 50px;
+}
```

Comment:
Please ensure that You stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).


Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/170848278-40e39c70-c4fc-4601-8157-5e33190cbc79.png) | ![image](https://user-images.githubusercontent.com/10332499/170848258-ffeeb23a-a1bc-4a1e-aeac-74d192c6be37.png)


Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/170848346-8f1a501d-8fe6-43d2-a8fc-b60db10697b2.png) | ![image](https://user-images.githubusercontent.com/10332499/170848333-844bb0bc-ec24-49b0-9d46-6aaac9d6d429.png)


Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/170848370-e4c88ca6-f7b4-4610-9e81-309dc7ef8725.png) | ![image](https://user-images.githubusercontent.com/10332499/170848375-09fb3614-9fda-4720-81d7-4b18764c42f7.png)